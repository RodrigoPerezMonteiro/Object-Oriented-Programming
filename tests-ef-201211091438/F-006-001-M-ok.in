7
3
All Pizza
2
carne de porco
300
8
1
3
carne de porco
0
0
0
