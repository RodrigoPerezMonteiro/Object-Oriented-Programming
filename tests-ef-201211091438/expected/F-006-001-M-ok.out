Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Identificador do restaurante (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do alimento (nome): Quantidade do alimento (inteiro): Preço do prato do dia (inteiro): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: NOVEG|carne de porco|8|0|UNAVAILABLE
Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): carne de porco|4200
Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 
